mixins/input.jsx

```diff
@@ -3,7 +3,7 @@ module.exports = {
     var self = this;
     var valid = true;
     fields.forEach(function(field) {
-      if (self.props.error[field] || !self.state.values[field]) {
+      if (self.props.error[field] || !self.state.values[field].trim()) {
```

Comment:
It's good practice before using trim to first make sure it's not false.

undefined.trim is a common console error.

So should look something like:

```
if (self.props.error[field] || !self.state.values[field] || !self.state.values[field].trim()) {
```